demo-notebooks/pymongo_demo.ipynb

```diff
@@ -0,0 +1,1196 @@
+    "    print(object_id)\n",
+    "\n",
+    "# can be done this way but do not want to print out all uuids\n",
+    "# for document in manifest_collection.find():\n",
+    "#     object_id = document['uuid']\n",
+    "#     print(object_id)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "0368eeac-bf87-4ec9-bdc4-9fd24c572297",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "25\n",
+      "['a6d713cc56c211f0bee30242ac19000b', 'a718bb1056c211f0bee30242ac19000b', 'a74ac5ec56c211f0bee30242ac19000b', 'a77a5d7a56c211f0bee30242ac19000b', 'a7b9d31a56c211f0bee30242ac19000b', 'a7eed9f256c211f0bee30242ac19000b', 'a8237cac56c211f0bee30242ac19000b', 'a857267e56c211f0bee30242ac19000b', 'a885ae4a56c211f0bee30242ac19000b', 'a8c1a33256c211f0bee30242ac19000b']\n"
+     ]
+    }
+   ],
+   "source": [
+    "# another example to get uuids by using the distinct function to get all uuids (trivial example to show some different functions,\n",
+    "# since all uuids are distinct)\n",
+    "manifest_ids = manifest_collection.distinct('uuid')\n",
+    "print(len(manifest_ids))\n",
+    "print(manifest_ids[0:10])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "814e9a7c-256a-43e9-b595-928e6fa883d4",
+   "metadata": {},
+   "source": [
+    "# Less trival examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "d9e39ff7-9ec0-42ac-9ab9-0128c48794ce",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "There are 3 out of 25 documents that satisfy the above query\n",
+      "Those documents' uuid's are \n",
+      "[{'_id': ObjectId('6864538b25e2fb1b531e4dfa'), 'uuid': 'a939dafa56c211f0bee30242ac19000b', 'basket_type': 'ch10', 'ch10name': '652200104150842', 'max_altitude': 11850, 'max_speed': 1434, 'fuel_burned': 6684}, {'_id': ObjectId('6864538c25e2fb1b531e4e03'), 'uuid': 'a9fd49b856c211f0bee30242ac19000b', 'basket_type': 'ch10', 'ch10name': '652200104150842', 'max_altitude': 12900, 'max_speed': 1424, 'fuel_burned': 7248}, {'_id': ObjectId('6864538f25e2fb1b531e4e15'), 'uuid': 'ab98588a56c211f0bee30242ac19000b', 'basket_type': 'ch10', 'ch10name': '652200104150842', 'max_altitude': 15000, 'max_speed': 1380, 'fuel_burned': 6378}]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# getting all documents' uuid's in metadata_collection that follows the query below\n",
+    "# max_altitude > 10_000\n",
+    "# max_speed < 1_450\n",
+    "# fuel_burned >= 6_000 and <= 8_000\n",
+    "query = {\"max_altitude\": {\"$gt\": 10_000}, \"max_speed\": {\"$lt\": 1_450}, \"fuel_burned\": {\"$gte\": 6_000, \"$lte\": 8_000}}\n",
+    "\n",
+    "# querying data like this is the main purpose of using a mongo db\n",
+    "# put the data into its own bucket and use weave for getting parent uuids, seeing what files in the bucket, etc.\n",
```

Comment:
Upload baskets with metadata to a pantry, and query the metadata using mongo.

Instead of "put the data into its own bucket and use weave for getting parent uuids, seeing what files in the bucket, etc.\n","